.gitignore

```diff
@@ -135,3 +135,4 @@ dmypy.json
 
 # other
 static/
+aws/
```

Comment:
Could you let us know why what we use **aws/** for?

Also, why do we have [awscliv2.zip](https://github.com/the-deep-nlp/core-server/pull/4/files#diff-8e0b2e2bfbff90d779612eea09a6e1ce95826ec9d7eb9055a8800d3ba1ee0f33) in the commit?

Reply:
@thenav56 I used it for installing aws client in the docker container. But now I have managed to install it from poetry and no more aws file will be generated.